docs/products/loader.mdx

```diff
@@ -36,3 +36,11 @@ import { Playground, PropsTable } from 'docz'
     <i class="dci dci-loader mi-spin dci-2x"></i>
 </span>
 </Playground>
+
+### Inline loader
+
+<Playground>
+<div class="inline-loading-container">
```

Comment:
Instead of adding a new container class for the inline loader, we can remove the fixed width for the loader-container. And inline would work with it